src/Provider/AbstractProvider.php

```diff
@@ -217,12 +227,8 @@ public function getAccessToken($grant = 'authorization_code', $params = [])
 
                 break;
         }
-
-        if (isset($result['error']) && ! empty($result['error'])) {
-            // @codeCoverageIgnoreStart
-            throw new IDPException($result);
-            // @codeCoverageIgnoreEnd
-        }
+        $checker = $this->errorChecker;
+        $checker($result);
```

Comment:
And I don't catch what the real getter/setter you mean is. Can you explain it please?